### ISSUE TITLE
SOCKS5 does not work in 12.0.0.

### ISSUE THREAD

**Issue by FabrizioRomanoGenovese** [windkh/node-red-contrib-telegrambot#263]
I just upgraded telegrambot to 12.0.0 and I get this error:
```
EFATAL: Error: connect ETIMEDOUT [ip]:[port]"

Timeout connecting to server. Maybe proxy blocked polling. Trying again.
```

telegrambot works normally with SOCKS5 proxy disabled. I've tried downgrading telegrambot but I ran into issues, most likely related to dependencies I guess, so I'm stuck with 12.0.0.

Any suggestions?

**Comment by windkh** [comment]
Hm strange, did it work with the previous version?

**Comment by windkh** [comment]
Have you tried to recreate the configuration node with the new version ?

**Comment by FabrizioRomanoGenovese** [comment]
It worked with the previous version, yes. It was 11.x.0, but I don't remember exactly what x was. I didn't try  yet to delete and recreate the bot node. I can try later today, but I have the feeling it won't help.

Does version 12 maybe use some newer version of the dependency providing SOCKS5? If so, maybe this newer version has breaking changes? Atm it just seems to constantly timeout. In the meantime I tested via commandline and my SOCKS5 server works fine.

**Comment by windkh** [comment]
@FabrizioRomanoGenovese sorry for delay, I was on vacation.
In version 12 I updated socks-proxy-agent to 7.0.

I will try to reproduce it with my local go-socks-proxy docker image as soon as I am home.

**Comment by windkh** [comment]
should be fixed in version 14.0.0
